tck/jax-rs/pom.xml

```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.tomee</groupId>
+        <artifactId>tck</artifactId>
+        <version>10.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>jax-rs-tck</artifactId>
+    <name>TomEE :: TCK :: Jakarta-Restfull-WS-TCK</name>
+
+    <properties>
+        <junit.jupiter.version>5.9.3</junit.jupiter.version>
+        <tomee-version>10.0.0-SNAPSHOT</tomee-version>
+    </properties>
+
+    <repositories>
+        <!--TODO: We use the staging repository of jakarta as the tck dependencies are not yet pushed to central. Remove this asap.-->
+        <repository>
+            <id>my-repo1</id>
+            <name>TCK-Staging-Repo</name>
+            <url>https://jakarta.oss.sonatype.org/content/groups/staging</url>
+        </repository>
+
+    </repositories>
+
+    <profiles>
+        <profile>
+            <id>record-signature</id>
```

Comment:
Can we split signature tests and actual tck (like it is done for bval for example) instead of having profiles here? (I think, that we can drop the profiles as well (similar to the other tck's run as part of the maven build))

Reply:
I removed the profiles and defined a build for the tests only.